bcgov/Unity
# Dev To Main Release 2024-02-22

QA Testable:

12241 - (Bugfix) Fix default mapping for project name on intake
23049 - (Bugfix) Pager reset in applications list on refresh
22958 - (Bugfix) Applications appear to move around the application list view (Updated, already with QA)
13501 - (Bugfix) Project info screen label updates
15444 - (Bugfix) Due Date / Past Date can be typed in and saved - Linked to 22937
13515 - Mapped fields that are missing from the Manage Columns dropdown
22937 - Update which fields are to remain editable after approval - Linked to 15444
22988 - Role (duty) shall be displayed in the Assignee column (Updated, already with QA)
12794 - Ability to filter Sector NAICS List of Value per tenant via database setting
22888 - Alphabetize Manage Columns List
